### ISSUE TITLE
Add metro-tile.png

### ISSUE THREAD

**Issue by Kroc** [h5bp/html5-boilerplate#1136]
See: https://blogs.msdn.com/b/ie/archive/2012/06/08/high-quality-visuals-for-pinned-sites-in-windows-8.aspx?Redirected=true

I have recently implemented this with NoNonsenseForum and I have learnt a couple of things:
- Windows8 canonicalises the colour, so the colour you give is not necessary the colour you get; therefore the image you provide _must_ have a transparent background
- Reusing apple-touch-icon is a no-go. Background must be transparent, and the logo should nearly fill the whole image because Metro provides its own spacing for the image
- _THE PNG MUST BE 32-BIT_ I've discovered that there's a bug in the Windows 8 Preview Release in that it won't render 8-bit, indexed PNGs for the metro tile. You _cannot_ use a PNG8+Alpha. When you use image optimisation tools like ImageOptim.app, they often reduce the colour depth where possible. You can manually optimise the image using particular command line switches to keep the bit-depth

In Summary:
- Create a 144x144 image of the H5BP logo, filling all of the canvas (no, or little padding), with a transparent background
- Save this image as a 32-bit PNG and optimise manually without reducing colour-depth, naming it whatever you want, but I call for "metro-tile.png" as standard
- Add the HTML metas to apply the tile and colour

Here's an image of what it looks like for my project, NoNonsenseForum:
http://camendesign.com/_stuff/metro-tile1.png - pin menu in IE
http://camendesign.com/_stuff/metro-tile2.png - Metro tile on the Start Screen

Edit: Oh, and you can’t use shorthand colour notation (i.e. "#222") for the colour, it has to be the full form ("#222222").
